project_recalculate/models/project_project.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
```

Comment:
It's possible for code files to be distributed isolated from their original package.
So IMO it's important to have at least have a short mention of the authors and license.
Though it's not a closed discussion; you might want to have a look at https://github.com/OCA/maintainer-tools/pull/91